ner/src/main/lbj/LbjTagger.lbj

```diff
@@ -207,6 +219,19 @@ discrete% Affixes(NEWord word) <-
 	}
 }
 
+discrete% AffixesZH(NEWord word) <-
+{
+	if(ParametersForLbjCode.currentParameters.featuresToUse.containsKey("Affixes")
+	&& ParametersForLbjCode.currentParameters.language.equals("zh"))
+	{
+  		int N = word.form.length();
+	  	for (int i = 1; i <= 2; ++i)
+    		if (word.form.length() > i) sense "ZH-p|" : word.form.substring(0, i);
+  		for (int i = 1; i <= 2; ++i)
+    		if (word.form.length() > i) sense "ZH-s|" : word.form.substring(N - i);
+	}
+}
+
 
```

Comment:
Oh this would be used only when the language is `ParametersForLbjCode.currentParameters.language.equals("zh")`; that's why it's not affected, I think ...